.env.example

```
@@ -19,6 +19,10 @@ VITE_VAPID_PUBLIC_KEY="your-vapid-public-key"
 # Feature Flags
 # Enable activity push notification fallback (true/false)
 VITE_ENABLE_ACTIVITY_PUSH_FALLBACK="false"
+# Enable native haptics path (defaults to web-first/no native when false)
+VITE_ENABLE_NATIVE_HAPTICS="false"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)